### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by Atefeh-Behroozi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
+node_modules
+=======
 /node_modules/
 .env
+>>>>>>> dfc20d92b8f8a94d25d8d3067e24e1c4d470505f
```
Comment (YurijPaharev) [inline .gitignore]: seems like the old problem accurred, please fix

File: frontend/src/pages/login/LoginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+const LoginPage: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [rememberMe, setRememberMe] = useState(false);
+  
+
+  const handleLogin = () => {
+    console.log('Email:', email);
+    console.log('Password:', password);
+    console.log('Remember Me:', rememberMe);
```
Comment (YurijPaharev) [inline frontend/src/pages/login/LoginPage.tsx]: Please do not leave console.logs even for a starting code

File: frontend/src/pages/login/LoginPage.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    <Container>
+      
+        <img src={logo} alt="Image" style={{ width: '140px', height: '30px', top: '40px', left:'76px' }} />
+      
+      <Grid container justifyContent="center" alignItems="center" style={{ height: '100vh' }}>
+        <Grid item xs={12} sm={6}>
+          
+            <Typography style={{
+              
+              font: 'Poppins',
```
Comment (YurijPaharev) [inline frontend/src/pages/login/LoginPage.tsx]: The code in this page is very poorly structured, I will try making an automated tool for that, but for now, please format it properly

File: package.json [hunk 4]
```diff
@@ -21,9 +21,7 @@
     "cross-env": "^7.0.3"
   },
   "dependencies": {
-    "firebase": "^10.4.0",
     "i": "^0.3.7",
-    "npm": "^10.1.0",
-    "react-hook-form": "^7.46.1"
+    "npm": "^10.1.0"
```
Comment (YurijPaharev) [inline package.json:25]: I think you can remove all of these dependencies

**Review by YurijPaharev — CHANGES_REQUESTED** [review]
Good work, please go through my comments and get rid of any inconsistencies

File: .gitignore (hunk 1 above)
Comment (Atefeh-Behroozi) [reply .gitignore]: Ok, I fix it now.

File: frontend/src/pages/login/LoginPage.tsx (hunk 3 above)
Comment (Atefeh-Behroozi) [reply frontend/src/pages/login/LoginPage.tsx]: My prettier formatter didn't work. I fix it now.

**Review by YurijPaharev — APPROVED** [review]

**Pull request merged**